src/DotNetProjectFile.Analyzers/Analyzers/MsBuild/ExcludePrivateAssetDependencies.cs

```diff
@@ -29,14 +30,9 @@ private static bool ShoudBePrivateAssets(PackageReference reference)
             return false;
         }
 
-        if (NuGet.Packages.All.TryGet(reference.Include) is { } pkg)
```

Comment:
You want to drop the manually maintained list for this analyzer?

Reply:
I do not think its add any value now we can access the nuspec files. We need those for licenses anyway, so they are one way or the other needed and cached for later usage. We could consider adding some extra data to it, that we want to define (like analyzers being language specific and or general purpose)